pkg/sql/pg_catalog.go

```diff
@@ -1166,7 +1165,7 @@ CREATE TABLE pg_catalog.pg_proc (
 	pronargs INT,
 	pronargdefaults INT,
 	prorettype OID,
-	proargtypes STRING,
+	proargtypes OID[],
 	proallargtypes STRING[],
```

Comment:
I just checked https://www.postgresql.org/docs/9.6/static/catalog-pg-proc.html

`proallargtypes` should be an `oid[]` ,and `proargtypes` should be an `oidvector`.